lib/sass_spec/test.rb

```diff
@@ -253,6 +261,7 @@ def handle_unexpected_pass!(test_case, options)
       return false
     end
 
+    return false if options[:probe_todo] && test_case.todo?
```

Comment:
I think we should define TestCase#probe_todo? and it should evaluate to `options[:probe_todo] && (todo? || warning_todo?)`